connect/src/com/telenor/connect/ConnectSdk.java

```diff
@@ -435,12 +435,20 @@ public static String getRedirectUri() {
         ArrayList<String> locales = new ArrayList<>();
         if (ConnectSdk.getLocales() != null && !ConnectSdk.getLocales().isEmpty()) {
             for (Locale locale : ConnectSdk.getLocales()) {
-                locales.add(locale.toString());
-                locales.add(locale.getLanguage());
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
Suggestion for improved readability:
locales.add(Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP ? locale.toLanguageTag() : locale.toString());

Reply:
Done